### ISSUE TITLE
Nuxt

### ISSUE THREAD

**Issue by khyoz** [directus/sdk-js#131]
# 🚀 How to install Directus SDK-JS with Nuxt ?

### Description
I try to follow the doc, but none of the methods seems to run

### Describe the solution you'd like
Can you provide an example how to start, with the filenames where to initialize the sdk ?

**Comment by WoLfulus** [comment]
is there any specific problems, like errors or something? what is not working? what you were expecting? 
can you provide a snippet of what you're trying to do, and what version you have installed?

**Comment by khyoz** [comment]
I try to use sdk-js in an nuxt project to access latest version of directus db. I try to follow the doc [Globally available](https://github.com/directus/sdk-js/blob/master/recipes/frameworks/Vue.md) or to use it as a component locally, but I always get errors. I don't know where to put the code `// main.js
import Vue from "vue";
import DirectusSDK from "@directus/sdk-js";

const client = new DirectusSDK({
    url: "https://demo-api.directus.app"
});

Object.defineProperties(Vue.prototype, {
    $client: { value: client }
});`
In which file in nuxt ? I tried as a plugin, no success, in a component, no success...
And there was written in the doc, if the doc for a recipe is missing, open a PR....
That's what I did
Documentation is not always easy for beginners and I didn't find examples on the net about that.
thanks for your support.
